tests/lib/snaps/generic-consumer/meta/snap.yaml

```diff
@@ -0,0 +1,6 @@
+name: generic-consumer
+version: 1.0
+apps:
+    cmd:
+        command: bin/cmd
+        plugs: [<interface-name>]
```

Comment:
Can you please use @INTERFACE@ instead. It shouts (at least for me) way more as the part that is being replaced. Ideally please also rename the file to snap.yaml.in so that it is fully obvious.